system/doc/system_principles/misc.xml

```diff
@@ -0,0 +1,182 @@
+	  subsequent releases.
+	</p>
+	<p>
+	  Loading on previous releases is <em>not</em> supported.
+	</p>
+      </item>
+      <tag>APIs</tag>
+      <item>
+	<p>Compatible between releases.</p>
+      </item>
+      <tag>Compiler Warnings</tag>
+      <item>
+	<p>New warnings may be issued between releases.</p>
+      </item>
+      <tag>Command Line Arguments</tag>
+      <item>
+	<p>Incompatible changes may occur between releases.</p>
+      </item>
+      <tag>OTP Build Procedures</tag>
+      <item><p>Incompatible changes may occur between releases.</p></item>
+    </taglist>
+    <p>
+      Under certain circumstances incompatible changes might be
+      introduced even in parts of the system that should be compatible
+      between releases. Things that might trigger incompatible changes
+      like this are:
+    </p>
+    <taglist>
+      <tag>Security Issues</tag>
+      <item>
+	<p>
+	  It might be necessary to introduce incompatible changes
+	  in order to solve a security issue. This kind of
+	  incompatibility might occur in a patch.
+	</p>
+      </item>
+      <tag>Bug Fixes</tag>
+      <item>
+	<p>
+	  We will not be bug compatible. A bug fix might introduce
+	  incompatible changes. This kind of incompatibility
+	  might occur in a patch.
+	</p>
+      </item>
+      <tag>Severe Previous Design Issues</tag>
+      <item>
+	<p>
+	  Some parts of OTP were designed a very long time ago and
+	  did not necessarily take today's computing environments into
+	  account. In some cases the consequences of those design
+	  decisions are too severe. This may be performance wise,
+	  scalability wise, etc. If we deem the consequences too
+	  severe, we might introduce incompatible changes. This kind
+	  of incompatibility will not be introduced in a patch, but
+	  instead in the next release.
+	</p>
+      </item>
+    </taglist>
+    <p>
+      Peripheral, trace, and debug functionality is at greater
+      risk of being changed in an incompatible way than functionality
+      in the language itself and core libraries used during operation.
+    </p>
+  </section>
+
+  <section>
+    <marker id="deprecation"/>
+    <title>Deprecation</title>
+    <p>
+      An API is deprecated when a new API is introduced that
+      is preferred to be used instead of the old API that is being
+      deprecated. The deprecation does <em>not</em> imply removal
+      of the API unless an upcoming removal is explicitly stated
+      in the deprecation.
+    </p>
+    <p>
+      The deprecated API will be documented as deprecated, and
+      compiler warnings will be issued, when appropriate, as
+      early as possible. That is, the new preferred API will
+      appear at the same time as the deprecation is issued.
+    </p>
+  </section>
+
+  <section>
+    <marker id="removal"/>
+    <title>Removal</title>
+    <p>
+      When it becomes necessary to remove legacy solutions they
+      are always phased out giving time for users to adopt.
```

Comment:
A bit weird to read. Maybe something like?

"Legacy solutions may eventually need to be removed. In such cases, they will be phased out on a long enough time period to give users the time to adapt."